registry/README.md

```diff
@@ -118,6 +118,60 @@ Note: ```quilt login``` is its own test that the server is working.  You can now
 
 To browse the catalog using a web browser, enter this location into your web browser: http://localhost:3000
 
+# Run a Private Registry in AWS
+
+## AWS Resources
+Running a Quilt Registry in AWS requires the following services:
+* a Postgres database. You can find instructions for setting up Postgres in RDS here: [Create a Postres Database in RDS](https://aws.amazon.com/rds/postgresql/)
+* an S3 bucket
+* an EC2 instance to run the registry, authentication service and catalog. Add the following policies to the security group: Postgres, HTTP/HTTPS, SSH and a custom TCP rule to enable port 5000.
+* an Elastic Load Balancer (ELB) to terminate SSL connections to the registry and catalog
+
+Once the resources have been created, ssh into the EC2 instance and configure the environment and run the services via Docker. You can make it easier to connect to your registry by creating a new DNS record to point to the EC2 instance's external IP address (e.g., quilt.yourdomain.com).
+
+```bash
+ssh ec2-user@quilt.yourdomain.com
+mkdir env
+sudo yum install docker
+sudo service docker start
+```
+
+## Configure your environment
+Edit the file env/registry and enter the following lines:
+```
+AWS_ACCESS_KEY_ID=<YOUR_CREDENTIALS_HERE>
+AWS_SECRET_ACCESS_KEY=<YOUR_CREDENTIALS_HERE>
+QUILT_SERVER_CONFIG=prod_config.py
+ALLOW_INSECURE_CATALOG_ACCESS=True
+SQLALCHEMY_DATABASE_URI=postgresql://$DB_USER:$DB_PASSWORD@$DB_HOST/$DB_DATABASE
+S3_HOST=s3
+REGISTRY_HOST=quilt.yourdomain.com
+REGISTRY_URL=http://${REGISTRY_HOST}:5000
+STRIPE_KEY=NOSTRIPE
+UWSGI_HOST=localhost
+UWSGI_PORT=9000
+NGINX_PORT=80
```

Comment:
Also needs CATALOG_URL, QUILT_SECRET_KEY, QUILT_DEFAULT_SENDER, SMTP_HOST, SMTP_USERNAME, and SMTP_PASSWORD